cpp/models/abm/simulation.h

```diff
@@ -30,55 +30,56 @@ namespace abm
 {
 
 /**
- * run the simulation in discrete steps, evolve the world and report results.
+ * @brief Run the Simulation in discrete steps, evolve the World and report results.
  */
 class Simulation
 {
     using ResultVector = Eigen::Matrix<int, Eigen::Index(InfectionState::Count), 1>;
 
 public:
     /**
-     * Create a simulation.
-     * @param t the starting time of the simulation
-     * @param world the world to simulate
+     * @brief Create a simulation.
+     * @param[in] t The starting time of the Simulation.
+     * @param[in,out] world The World to simulate.
```

Comment:
Not sure about the "out" here. The world is moved from the argument to the internal world object. Sure, it changes, but maybe this is a bit different. @mknaranja What do you think?

Reply:
Removed it.